tests/schemas/api_v3/transactionsDryRun.schema.js

```diff
@@ -77,5 +84,8 @@ module.exports = {
 	dryrunTransactionInvalidResponseSchema: Joi.object(
 		dryrunTransactionInvalidResponseSchema,
 	).required(),
-	metaSchema: Joi.object().required(),
+	metaSchema: Joi.object().optional(),
+	goodRequestSchemaFortransactionsDryRun: Joi.object(
+		goodRequestSchemaFortransactionsDryRun,
+	).optional(),
```

Comment:
Should this be `.required` since data and meta are always returned.
https://github.com/LiskHQ/lisk-service/blob/development/services/blockchain-indexer/shared/dataService/business/transactionsDryRun.js#L28